backend/applications/models.py

```diff
@@ -0,0 +1,105 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from django_cryptography.fields import encrypt
+
+from applications.enums import ApplicationStatus
+from companies.models import Company
+from utils.models import HistoricalModel, UUIDModel
+
+
+class Application(HistoricalModel, UUIDModel):
+    company = models.ForeignKey(
+        Company,
+        on_delete=models.CASCADE,
+        related_name="applications",
+        verbose_name=_("company"),
+    )
+    status = models.CharField(
+        max_length=64, verbose_name=_("status"), choices=ApplicationStatus.choices
+    )
+
+    invoicer_name = models.CharField(
+        blank=True,
+        null=True,
```

Comment:
Avoid `null=True` on `CharField`s and `TextField`s. Otherwise you'll end up having two possible values for "no data"; `NULL` or an empty string. (refs. https://docs.djangoproject.com/en/3.2/ref/models/fields/#django.db.models.Field.null)

This applies to the following fields on this model:
- invoicer_name
- invoicer_email
- invoicer_phone_number

Reply:
Good point, I'll change!